src/routes/jira/connect/enterprise/app/jira-connect-enterprise-app-post.ts

```diff
@@ -33,13 +33,11 @@ export const JiraConnectEnterpriseAppPost = async (
 			installationId: installation.id
 		});
 
-		res.status(201).json({
-			message: `Successfully added GitHub Server App for ${installation.id}`,
-			data: githubServerApp
-		});
+		res.status(200).send({ success: true, data: githubServerApp });
 
 		req.log.debug("Jira Connect Enterprise App added successfully.");
 	} catch (error) {
+		res.status(200).send({ success: false, message: "Failed to create GitHub App." });
```

Comment:
Not sure is there a way you can double check this? 
If we do `res.status(xxx).send(xxx)` here, will it conflict with whatever error router handler down the track when we `return next(new Error(xxx))`?